extra/string-extra/src/StringExtension.php

```diff
@@ -21,11 +22,17 @@ public function getFilters()
     {
         return [
             new TwigFilter('u', [$this, 'createUnicodeString']),
+            new TwigFilter('slug', [$this, 'createSlug']),
         ];
     }
 
     public function createUnicodeString(?string $text): UnicodeString
     {
         return new UnicodeString($text ?? '');
     }
+
+    public function createSlug(?string $string, ?string $separator = '-', string $locale = null): string
```

Comment:
should be ` public function createSlug(string $string, string $separator = '-', string $locale = null): string` to be consistent with the `AsciiSlugger` signature.

And it returns a `AbstractUnicodeString` instance. Not sure if we should always return a string or the instance.